cost/azure/savings_realized/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v3.7
+
+- Issue with `bill_source_expressions` fixed, in case that was empty, policy breaks on runtime since `or` condition doesn't support empty expressions, added an extra validation to get rid of that condition if `bill_source_expressions` is empty.
```

Comment:
This change is just a more thorough explanation of what is happening under the hood. This is not what the CHANGELOG should contain.

Imagine you are a user using the policy. What problem would you experience? Describe that in terms of what the user would experience, not what the code is doing.

Reply:
Specifically, what are the conditions in which the policy would fail from a user's point of view? What does the `or` condition, and its emptiness, represent in terms of what the user is trying to do?

Reply:
Description updated, I think now it's more related with the customer's point of view, anyway please let me know if you see any improvement to be applicable.